### PR TITLE
Fix sed command's wrong behavior under MacOS

### PR DESCRIPTION
**Pull request by ppfish45**
`sed` command will give "invalid command code ." error under MacOS if the extension of backup is not specified.

Therefore, I added a OS check in `build_standalone_docker.sh` so that this script can execute correctly under MacOS.

### CODE REVIEW

**Comment by dylan-fan** [comment]
can you give me your email or wechat?  We will send you some gift for your contribution

File: standalone-deploy/build_standalone_docker.sh [hunk 1]
```diff
@@ -26,9 +26,15 @@ rm -rf init.sh requirments.txt requirements.txt
 
 cp ../requirements.txt ./docker/python
 
-#sed -i "s/'user':.*/'user': 'fate_dev',/g" ./fate_flow/settings.py
-#sed -i "s/'passwd':.*/'passwd': 'fate_dev',/g" ./fate_flow/settings.py
-#sed -i "s/'host':.*/'host': 'mysql',/g" ./fate_flow/settings.py
```
Comment (zengjice) [inline standalone-deploy/build_standalone_docker.sh]: This piece of code is obsolete and does not need to be reopened.

File: standalone-deploy/build_standalone_docker.sh [hunk 2]
```diff
@@ -26,9 +26,15 @@ rm -rf init.sh requirments.txt requirements.txt
...
+if [ `uname -s` == "Darwin" ]; then
+    sed -i ".bak" "s/'user':.*/'user': 'fate_dev',/g" ./fate_flow/settings.py
```
Comment (zengjice) [inline standalone-deploy/build_standalone_docker.sh]: Thank you for your suggestion, but here can use 'sed -i "" "s/'user':.*/'user': 'fate_dev',/g" ./fate_flow/settings.py' to keep linux and mac consistent. We will open a new pull request.

Comment (ppfish45) [reply standalone-deploy/build_standalone_docker.sh]: Thanks so much for resolving this problem with better solution =)

**Comment by ppfish45** [comment]
> can you give me your email or wechat? We will send you some gift for your contribution

Hi, my wechat is ppfish361429651 and my email is ppfish.45@gmail.com

Thanks a lot!!

**Pull request closed without merging**